<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIauthentication</class>
 <widget class="QMainWindow" name="UIauthentication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UIauthentication</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>621</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border-radius: 20px;</string>
    </property>
    <widget class="QLabel" name="labelLogin">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Calligraphy</family>
       <pointsize>26</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Login Form</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_login">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>460</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(67, 67, 100);
background-color: rgb(113, 111, 172);
border-radius: 10px;
color: rgb(223, 223, 223);</string>
     </property>
     <property name="text">
      <string>Sign In</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(81, 81, 121);
border-radius: 10px;
border-bottom: 1px solid #ffffff;
color: rgb(223, 223, 223);</string>
     </property>
     <property name="placeholderText">
      <string>USERNAME</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>380</y>
       <width>321</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(81, 81, 121);
border-radius: 10px;
border-bottom: 1px solid #ffffff;
color: rgb(223, 223, 223);</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>PASSWORD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>530</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(67, 67, 100);
background-color: rgb(173, 52, 36);
border-radius: 10px;
color: rgb(223, 223, 223);</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_signIn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>460</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(67, 67, 100);
border-radius: 10px;
color: rgb(223, 223, 223);</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
